astro/src/content/blog/fusionauth-re-invent-2023.mdx

```diff
@@ -0,0 +1,59 @@
+
+We're also looking forward to some AWS re:Invent sessions. 
+
+Exciting ones include 
+- SEC209 | Modernize authorization: Lessons from cryptography and authentication
+- DAT333 | Building highly resilient applications with Amazon DynamoDB
+- SEC322 | Integrate apps with Amazon Cognito and Amazon Verified Permissions
+- BOA209 | Build verifiable and effective application authorization in 40 minutes
+
+Of course, it isn't all education. There's also fun to be had!
+
+One fun thing to do is join [Corey Quinn's annual nature walk](https://www.lastweekinaws.com/blog/aws-re-invent-what-you-actually-need-to-know-before-you-go/) where he skewers Expo vendors. This is happening on Wednesday at 1pm.
+
+There are also numerous parties. If you are attending AWS, chances are at least one vendor has emailed you with an invite. This Twitter account, [@reInventParties](https://twitter.com/reInventParties/), is a great way to discover additional opportunities for fun.
+
+Finally, if nature is your thing, we've heard [Red Rock Canyon](https://www.blm.gov/programs/national-conservation-lands/nevada/red-rock-canyon) has good hiking and climbing.
+
+## Person On The Street
+
+Have you ever wanted to be one of those people in Vegas that they walk up to and ask questions to win prizes? Well, you are in luck because from the moment my shoes hit the ground at re:Invent this year I will be recording all the action. I will be asking some hard-to-answer questions about Authentication, but when you get the answer right you will win a collector's set of FusionAuth trading cards. With each set, there will be an opportunity to win swag at our booth #152.
+
+![Alex with FusionAuth hat and trading cards behind him.](/img/blogs/fusionauth-re-invent-2023/reinvent-cards.png)
+
+## Meet up with FusionAuth
+
+If you want to meet up with the FusionAuth team at re:Invent this week, we're around and ready to chat. Find us in a couple of places:
+
+* We're having a Party on Wednesday, November 29 at [ROCKHOUSE](https://www.therockhousebar.com/). It's currently full, but please feel free to join the waitlist.
+* We have booth #152 in the Expo hall neare the walkway to Caesars Forum, so if you see Red Hat and Cockroach Labs, you are in the right place! There's no public map for the 2023 re:Invent Expo yet, but [here's the page](https://reinvent.awsevents.com/learn/expo/). Brave the madness and come say hi for a demo of biometric authentication, chat about the intricacies of the Authorization Code grant, or sign up for giveaways. You can also get VIP access to the aforementioned party at ROCKHOUSE.
+* If you want to meet with a team member to discuss how FusionAuth can help you in more detail, visit [our 2023 AWS re:Invent page](/aws-reinvent23) and click on **Schedule a Meeting**.
+* Finally, we'll be out and about as well. Send us a note via our [contact form](/contact) or post to us on [X](https://twitter.com/fusionauth) (yeah that used to say tweet). 
```

Comment:
Is anyone going to post on the FusionAuth account while there?  Should you ask people to follow FusionAuth as well?

Reply:
Do you mean the LinkedIn account? This is our twitter account. I don't like asking for follows but let them make that call.